UniAtHome/UniAtHome.WebAPI/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=.\\SQLEXPRESS;Database=UniAtHomeDatabase;Trusted_Connection=True;"
```

Comment:
We should decide what we going to do with connection string. Currenly I use local one.

Reply:
Today I'll create another settings file which will be gitignored. There we'll put our confidential info such as connection strings. Making it gitignored allows us to swap database to local one and back to the hosted one.